SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestLoadBalancingData/SpatialTestLoadBalancingData.cpp

```diff
@@ -141,75 +147,61 @@ void ASpatialTestLoadBalancingData::PrepareTest()
 		FinishStep();
 	});
 
-	constexpr float ActorReceiptTimeout = 5.0f;
-	AddStep(
-		TEXT("Retrieve actors on all workers"), FWorkerDefinition::AllWorkers, nullptr, nullptr,
-		[this](float) {
-			TargetActor = GetWorldActor<ASpatialTestLoadBalancingDataActor>(GetWorld());
-			RequireTrue(TargetActor.IsValid(), TEXT("Received main actor"));
+	AddStep(TEXT("Retrieve actors on all workers"), FWorkerDefinition::AllWorkers, nullptr, nullptr, [this](float) {
+		TargetActor = GetWorldActor<ASpatialTestLoadBalancingDataActor>(GetWorld());
+		RequireTrue(TargetActor.IsValid() && TargetActor->IsActorReady(), TEXT("Received main actor"));
```

Comment:
```suggestion
		RequireTrue(TargetActor.IsValid(), TEXT("Received main actor"));
```
In which case we should need to check `TargetActor->IsActorReady()` anymore